### PR TITLE
460 show shelfmarks in doc detail

### PR DESCRIPTION
**Pull request by kmcelwee**
Example of a multi-shelfmark doc: PGPID 444

### CODE REVIEW

File: geniza/corpus/templates/corpus/document_detail.html [hunk 1]
```diff
@@ -48,6 +48,10 @@ <h3 class="sr-only">
                         {# link ? #}
                     {% endfor %}
                 {% endif %}
+                {% if document.fragments.count > 1 %}
+                    <dt class="secondary">{% translate 'Shelfmark' %}</dt>
+                    <dd class="secondary">{{ document.shelfmark }}</dd>
+                {% endif %}
```
Comment (rlskoeser) [inline geniza/corpus/templates/corpus/document_detail.html:54]: per the linked designs shelfmark should be displayed before editor and should not be in the lighter gray color

File: geniza/corpus/tests/test_corpus_templates.py [hunk 2]
```diff
@@ -136,6 +136,16 @@ def test_editors(self, client, document, source, twoauthor_source):
         response = client.get(document.get_absolute_url())
         assertContains(response, "Editors")
 
+    def test_multiple_shelfmarks(self, client, document, join):
+        """Document detail template should show multiple shelfmarks"""
+        response = client.get(document.get_absolute_url())
+        # TODO: This is fragile, consider something better
+        assertNotContains(response, "Shelfmark", html=True)
```
Comment (rlskoeser) [inline geniza/corpus/tests/test_corpus_templates.py:143]: you're not actually using an html match here, but that would be more robust — why not include the full `<dt>` tag?

File: geniza/corpus/tests/test_corpus_templates.py [hunk 3]
```diff
@@ -136,6 +136,16 @@ def test_editors(self, client, document, source, twoauthor_source):
         response = client.get(document.get_absolute_url())
         assertContains(response, "Editors")
 
+    def test_multiple_shelfmarks(self, client, document, join):
```
Comment (rlskoeser) [inline geniza/corpus/tests/test_corpus_templates.py:139]: please add comments or separate into two tests to clarify that one _should_ display the shelfmark as a field and one should not

**Review by rlskoeser — CHANGES_REQUESTED** [review]
logic looks fine, but placement and styles are not quite right and the test should be fixed as you noted in your TODO

File: geniza/corpus/templates/corpus/document_detail.html (hunk 1 above)
Comment (kmcelwee) [reply geniza/corpus/templates/corpus/document_detail.html]: 1a3f626 ✅

File: geniza/corpus/tests/test_corpus_templates.py (hunk 2 above)
Comment (kmcelwee) [reply geniza/corpus/tests/test_corpus_templates.py]: ff40fa023ef5d9811b1622c51b2692d64423e05a ✅

File: geniza/corpus/tests/test_corpus_templates.py (hunk 3 above)
Comment (kmcelwee) [reply geniza/corpus/tests/test_corpus_templates.py:139]: 9f8c3ee ✅

**Review by rlskoeser — APPROVED** [review]

**Pull request merged**